test/phx_tools_web/live/index_test.exs

```diff
@@ -30,29 +30,26 @@ defmodule PhxToolsWeb.PhxToolsLive.IndexTest do
 
     test "user visits the page with Linux OS user agent", %{conn: conn} do
       updated_conn = put_req_header(conn, "user-agent", "Linux")
-      {:ok, landing_live, html} = live(updated_conn, "/")
+      {:ok, _landing_live, html} = live(updated_conn, "/")
 
       assert html =~ "The Complete Development Environment for Elixir and Phoenix"
       assert html =~ "Linux.sh"
-
-      assert has_element?(landing_live, "#macOS")
     end
 
     test "user visits the page with Mac OS user agent", %{conn: conn} do
       updated_conn = put_req_header(conn, "user-agent", "Mac OS X 10_5_7")
-      {:ok, landing_live, html} = live(updated_conn, "/")
+      {:ok, _landing_live, html} = live(updated_conn, "/")
 
       assert html =~ "The Complete Development Environment for Elixir and Phoenix"
       assert html =~ "macOS.sh"
```

Comment:
We don't want this anymore.
As you can see,
https://github.com/optimumBA/phx.tools/blob/8a846786cce721ce782d93b7d3d5513c33da9e15/priv/static/macOS.sh#L1C1-L1C13
is pretty useless for anyone wondering what the script code looks like.
Point to the `script.sh` file instead.